skietbaan_fe/src/components/AddMembersGroup.js

```diff
@@ -49,7 +57,7 @@ class AddMembersGroup extends Component {
       users: this.state.newArray,
       GroupIds: this.props.id
     };
-    fetch(BASE_URL+"/api/groups/postMember/", {
+    fetch(BASE_URL + "/api/groups/postMember/", {
```

Comment:
above it was `api/Groups/` so which one is it?

Reply:
different controller